docs/user-guide/configuration.rst

```diff
@@ -234,7 +234,19 @@ Playbook actions are loaded into Robusta using the ``playbookRepos`` Helm value.
 
 Robusta has a set of builtin playbooks.
 
-You can load extra playbook actions from git repositories:
+You can load extra playbook actions in two different ways from git repositories.
+
+1) Loading a git playbook by HTTPS:
+
+.. code-block:: yaml
+
+    playbookRepos:
+      # we're adding the robusta chaos-engineering playbooks here
```

Comment:
I would add a link in the comment to https://github.com/robusta-dev/robusta-chaos